notebook/static/tree/js/notebooklist.js

```diff
@@ -386,6 +386,7 @@ define([
         var root = $('<li/>').append(
             $("<a/>")
             .attr('href', root_url)
+            .attr('title','Link to root folder'+root_url)
```

Comment:
This presumably needs integrating with the translation machinery.

Also, it doesn't seem right to put the URL in the title. Again, I'm quite ignorant of accessibility concerns, but presumably screenreader tools have a way to inspect the URL of a link if the user wants that.

Reply:
This has been updated